zproject/jinja2/__init__.py

```diff
@@ -16,7 +16,7 @@ def environment(**options: Any) -> Environment:
     env = Environment(**options)
     env.globals.update(
         default_page_params={
-            "debug_mode": False,
+            "debug_mode": settings.DEBUG,
```

Comment:
Oh, it's convenient that we already had this plumbed.  I think the `analytics/views.py` code path needs a change as well ... though I think possibly just deleting the `debug_mode` line, since it should inherit `default_page_params` for any value it doesn't explicitly set.
```
$ git grep debug_mode
analytics/views.py:        debug_mode=False,
static/js/blueslip.js:    if (page_params.debug_mode) {
static/js/blueslip.js:    if (page_params.debug_mode) {
static/js/blueslip.js:    if (page_params.debug_mode) {
zerver/lib/home.py:        debug_mode=settings.DEBUG,
zerver/tests/test_home.py:        "debug_mode",
zproject/jinja2/__init__.py:            "debug_mode": False,
(zulip-py3-venv) tabbott@coset:~/zulip$ less analytics/views.py
```

Reply:
Yes, it's actually kind of confusing when `debug_mode` is still False when it should be True.